packages/instantsearch.js/src/lib/utils/__tests__/getAppIdAndApiKey-test.ts

```diff
@@ -20,4 +20,34 @@ describe('getAppIdAndApiKey', () => {
     expect(appId).toEqual(APP_ID);
     expect(apiKey).toEqual(API_KEY);
   });
+
+  it("fallsback to baseHeaders if headers doesn't exist", () => {
+    const searchClient = {
```

Comment:
Could we install v5 like the other versions too? That way it gives full confidence IMO